src/test_linked_list.py

```diff
@@ -0,0 +1,112 @@
+"""This tests the linked_list module."""
+import pytest
+
+INSTANTIATION_TABLE = [
+    ("new_linked_list"),
+]
```

Comment:
If your parametrization table only has one item in it, it's not worth having the table.